madmom/audio/cepstrogram.py

```diff
@@ -34,6 +37,7 @@ class applies some transformation (usually a DCT) on a spectrogram.
         one is instantiated with these additional keyword arguments.
 
     """
+
```

Comment:
My code formatter is doing this, I will try to change the behaviour. Else it will happen again for sure.